packages/google_atlas/lib/src/google_atlas.dart

```diff
@@ -2,6 +2,7 @@ import 'dart:async';
 
 import 'package:flutter/widgets.dart';
 import 'package:atlas/atlas.dart';
+import './utils/lat_lng_bounds_utils.dart';
```

Comment:
Let's not use relative imports... if necessary, create a `utils/utils.dart` where we can include all the `imports`, so here we only need to import `package:google_atlas/google_atlas.dart`

Reply:
as a matter of fact, my original code import package, but Felix suggested to use relative path

Reply:
@jorgecoca I don't think we should expose the utils publicly and I thought it was bad practice to use package imports that reference src